examples/undo/example.py

```diff
@@ -0,0 +1,96 @@
+# (C) Copyright 2008-2020 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+# -----------------------------------------------------------------------------
+# Copyright (c) 2008, Riverbank Computing Limited
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in enthought/LICENSE.txt and may be redistributed only
+# under the conditions described in the aforementioned license.  The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+# Thanks for using Enthought open source!
+#
+# Author: Riverbank Computing Limited
+# Description: <Enthought undo package component>
+# -----------------------------------------------------------------------------
+
+
+# Standard library imports.
+import logging
+
+# Enthought library imports.
+from pyface.api import GUI, YES
+from pyface.workbench.api import Workbench
```

Comment:
can you open an issue regd porting the undo example code from pyface workbench to pyface tasks?

Reply:
Opened #821